Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -260,6 +292,31 @@ - (OCTMessageAbstract *)addMessageWithText:(NSString *)text
     return messageAbstract;
 }
 
+- (void)addMessageCall:(OCTMessageCallEvent)event
+                  call:(OCTCall *)call
+          callDuration:(NSTimeInterval)duration
+{
+    NSParameterAssert(event);
+    NSParameterAssert(call);
+    DDLogInfo(@"OCTRealmManager: adding messageCall to call %@", call);
+
+    OCTMessageCall *messageCall = [OCTMessageCall new];
+    messageCall.callDuration = duration;
+    messageCall.callEvent = event;
+
+    OCTMessageAbstract *messageAbstract = [OCTMessageAbstract new];
+    messageAbstract.dateInterval = [[NSDate date] timeIntervalSince1970];
+    messageAbstract.sender = call.chat.friends.firstObject;
```

Comment:
In case if you were calling this is wrong. In that case `sender` property should be nil.